packages/api-key-stamper/src/utils.ts

```diff
@@ -21,21 +21,21 @@ export function convertTurnkeyApiKeyToJwk(input: {
   // Ensure that each of the constituent parts are sufficiently padded
   jwkCopy.d = hexStringToBase64urlString(uncompressedPrivateKeyHex);
 
-  // Manipulate x and y
-  const decodedX = base64urlToBuffer(jwkCopy.x!);
-  const paddedX = hexStringToBase64urlString(
-    uint8ArrayToHexString(new Uint8Array(decodedX)),
-    DEFAULT_JWK_MEMBER_BYTE_LENGTH
-  );
+  // // Manipulate x and y
```

Comment:
temporarily commenting this logic out in order to be able to reproduce the issue when converting turnkey API key to jwk